tomlkit/parser.py

```diff
@@ -424,9 +424,10 @@ def _parse_value(self) -> Item:
             return self._parse_basic_string()
         elif c == StringType.SLL.value:
             return self._parse_literal_string()
-        elif c == BoolType.TRUE.value[0]:
+        # peek one furhter then the value so we don't parse values that look like trueXXX or falseXXX
+        elif BoolType.TRUE.value.startswith(c) and self._peek(len(BoolType.TRUE.value) + 1) == BoolType.TRUE.value:
```

Comment:
Is `startswith` more efficient than `==` ? any other reason for changing this?

Reply:
I just find it more clear what the actual check is. The check could also just be completely removed, as it's included in the second term

Reply:
Yes, but I would prefer to keep it as a shortcut.

Reply:
Also remember to format the code with `black` before commit. It is integrated into `pre-commit` if you have installed it.

Reply:
Yeah sorry, I overlooked installing the `pre-commit` hooks. I've fixed those issues now and reverted the check to a simple equality `==` check